.circleci/config.yml

```diff
@@ -445,7 +453,14 @@ workflows:
   version: 2
   build-distros:
     jobs:
-      - go-fmt-and-vet
+      - lint-consul-retry:
+          filters:
+            branches:
+              ignore:
+                - /^pull\/.*$/ # only run link checker on non forks
```

Comment:
Since the repo is public now, should this be open for forks as well?

Reply:
ah yeah good point, I'll change that!